pennylane/_device.py

```diff
@@ -91,13 +89,12 @@ def _local_tape_expand(tape, depth, stop_at):
             # recursively expand out the newly created tape
             expanded_tape = _local_tape_expand(obj, stop_at=stop_at, depth=depth - 1)
 
-            new_prep.extend(expanded_tape._prep)
-            new_ops.extend(expanded_tape._ops)
+            new_ops.extend(expanded_tape.operations)
             new_measurements.extend(expanded_tape._measurements)
```

Comment:
```suggestion
            new_measurements.extend(expanded_tape.measurements)
```

Probably don't need that to use the private property either.

Reply:
there are a few spots that need updating, i'll get them all locally.